templates/alertmanager/alertmanager-dep.yaml

```
@@ -120,7 +120,9 @@ spec:
               mountPath: /etc/cortex-runtime-config
             - name: storage
               mountPath: "/data"
-              subPath: {{ .Values.alertmanager.persistence.subPath }}
+              {{- if .Values.alertmanager.persistentVolume.subPath }}
+              subPath: {{ .Values.alertmanager.persistentVolume.subPath }}
+              {{- end }}
```

Not critical, but the shorter way to write this is
```suggestion
              {{- with .Values.alertmanager.persistentVolume.subPath }}
              subPath: {{ . }}
              {{- end }}
```

---

sure but maybe we should rework the entire chart then, its kinda mixed between simple if's and scoped variables with the "with" statement

---

I love `with`